src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.util.regex.Pattern;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.validateNotNull;
+
+public final class StringValidators {
+
+    private StringValidators() {}
+
+    public static boolean validateStringNotBlank(String toTest, String location, Errors errs, String loggingContext) {
```

Comment:
Could drop 'String' from the method name here. It's in the StringValidators class and next max length method doesn't include 'String' in name.

Reply:
Removed from this and other methods in this class.